tests/update_reference_data.sh

```diff
@@ -133,6 +148,7 @@ then
   git status | grep "SPE1CASE1" && tests="spe11"
   git status | grep "SPE1CASE2" && tests="$tests spe12"
   git status | grep "SPE3CASE1" && tests="$tests spe3"
+  git status | grep "SPE1CASE2_2P" && tests="$tests spe1-2p"
```

Comment:
this generates the log message. check if file associated with this is in the update log, if so, append to commit message.